movement/movement.py

```diff
@@ -0,0 +1,366 @@
+            
+            if abs(correction_angle) > 45:
+                correction_angle = math.copysign(45, current_angle)
+
+            # negative correction_angle for rear wheels
+            FLServo.ChangeDutyCycle(get_DC_from_angle(math.copysign(1, -direction) * correction_angle + correct_dist_angle))
+            FRServo.ChangeDutyCycle(get_DC_from_angle(math.copysign(1, -direction) * correction_angle + correct_dist_angle))
+            BLServo.ChangeDutyCycle(get_DC_from_angle(math.copysign(1, direction-1) * correction_angle + correct_dist_angle))
+            BRServo.ChangeDutyCycle(get_DC_from_angle(math.copysign(1, direction-1) * correction_angle + correct_dist_angle))
+
+            drive_all(direction, *motor_speeds)
+
+            time.sleep(0.1)
+
+        """# Differential drive
+        print("Starting loop")
+        while True:
+            # stop
+            if front.distance < .20:
+                print("STOPPING: " + str(front.distance))
+                brake()
+                break
+                # cycle += 1
+                # if cycle == 2:
+                #     break
+
+                # target_dist = sensors[side].distance
+                # side = (side + 1) % 2
+
+                # if side == 0:
+                #     turn = turn_left_dime
+                # else:
+                #     turn = turn_right_dime
+
+                # while sensors[side].distance > target_dist:
+                #     turn(base_speed)
+
+            distance = sensors[side].distance
+            print(distance)
+            # in range correction
+            if distance > previous_dist:
+                speeds[side] = base_speed - 5
+                speeds[side-1] = base_speed + 5
+            if distance < previous_dist:
+                speeds[side] = base_speed + 5
+                speeds[side-1] = base_speed - 5
+
+            if distance == previous_dist:
+                speeds[side] = base_speed
+                speeds[side-1] = base_speed
+
+            # out of range correction
+            if distance < 0.15 and distance < previous_dist + .01:
+                speeds[side] = base_speed + 10
+                speeds[side-1] = base_speed - 10
+            if distance < 0.15 and distance > previous_dist:
+                speeds[side] = base_speed
+                speeds[side-1] = base_speed
+
+            if distance > 0.20 and distance > previous_dist - 0.01:
+                speeds[side] = base_speed - 10
+                speeds[side-1] = base_speed + 10
+            if distance > 0.20 and distance < previous_dist:
+                speeds[side] = base_speed
+                speeds[side-1] = base_speed
+
+            forward(speeds[0], speeds[1])
+
+            previous_dist = distance
+
+            time.sleep(0.1)"""
+
+    except KeyboardInterrupt:
+        brake()
+        pass
+
+    except Exception:
+        brake()
+        pass
+
+    finally:
+        cleanup_board()
+
+def cleanup_board():
```

Comment:
Would rename this to end_movement() to make the code in main more understandable.